packages/near-fast-auth-signer-e2e-tests/pages/DevicesPage.ts

```diff
@@ -0,0 +1,37 @@
+import { Page } from '@playwright/test';
+
+import { checkConditionUntilMet } from '../utils/async';
+
+class DevicesPage {
+  private page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  async getCount() {
+    const checkboxes = this.page.locator('input[type="checkbox"]');
+    return checkboxes.count();
+  }
+
+  async selectAndDelete() {
+    const checkFunction = async () => {
+      const count = await this.getCount();
+      return count > 0;
+    };
+    // Wait until it loads records and it is available for an action
+    await checkConditionUntilMet(this.page, checkFunction, 5, 1000);
```

Comment:
By what I understood this code is waiting for the checkbox to load. 

Can we replace this with: 

`await expect(this.page.locator('input[type="checkbox"]')).toHaveCount(5, {timeout: 20000});`

Or any other check around the existence of the checkboxes? So we can take advantage of the native methods of Playwright. 

Then we can clean up some code too, like: `getCount`, `checkConditionUntilMet` and `checkFunction`

Reply:
will prob need to increase timeout because this step actually requires some time to execute unfortunately..

Reply:
Sounds good, 20s it's just the value I tested locally, but on CI or ur machine it may need more time. 

My main concern was replace it with native playwright code